packages/node_modules/@ciscospark/widget-meet/src/components/call-active/index.js

```diff
@@ -21,8 +21,8 @@ export default function ActiveCall({
   onStopSendingVideo,
   localMediaStream,
   remoteMediaStream,
-  toPersonAvatar,
-  toPersonName
+  avatarImage,
```

Comment:
I'm glad you changed these to more agnostic naming instead of trying to use a room name as a `toPersonName`. Good call.